comp/api/api/apiimpl/api.go

```diff
@@ -16,7 +16,8 @@ import (
 	"github.com/DataDog/datadog-agent/comp/api/authtoken"
 	"github.com/DataDog/datadog-agent/comp/collector/collector"
 	"github.com/DataDog/datadog-agent/comp/core/autodiscovery"
-	"github.com/DataDog/datadog-agent/comp/core/flare"
+
+	// "github.com/DataDog/datadog-agent/comp/core/flare"
```

Comment:
Can we remove the comment?

Reply:
removed in https://github.com/DataDog/datadog-agent/pull/24045/commits/dd24bb32483d68bc40fbebc5a077395d8fa2a4a4